src/Ledger/Conway/Epoch.lagda

```diff
@@ -435,6 +437,8 @@ its results by carrying out each of the following tasks.
            ; pools = pState .pools ; delegatees = dState .voteDelegs }
         ⊢ ⟦ es , ∅ , false ⟧ ⇀⦇ govSt' ,RATIFIES⦈ fut'
       → ls ⊢ ss ⇀⦇ tt ,SNAP⦈ ss'
+      → _  ⊢ ⟦ utxoSt , acnt , dState , pState ⟧ ⇀⦇ e ,POOLREAP⦈
+             ⟦ utxoSt' , acnt' , dState' , pState' ⟧
```

Comment:
I don't know where to get `pp : PParams` from. In the Shelley spec it is an input to the EPOCH transition.